progress_go1.7.go

```diff
@@ -0,0 +1,16 @@
+//+build go1.7
+
+package mpb
+
+import "context"
+
+// New Progress instance, it orchestrates the rendering of progress bars.
+// It supports cancellation via Context.
+func NewWithCtx(ctx context.Context) *Progress {
+	p := New()
+	go func() {
+		<-ctx.Done()
+		p.Stop()
+	}()
+	return p
+}
```

Comment:
Thank you for your work. But using context for cancelation is de-facto standard. As far as I know, context package is available in Go 1.6, but under `golang.org/x/net/context` path. I don't like the idea of removing context package. As for `New(nil)`, I agree it is a bit weird, but I did this on purpose, so one thinks about cancelation beforehand. By the way I plan to make some significant changes and bump to v2, so maybe I'll make just `New()`.